src/BPEssentials/Utils/ChatUtils.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Collections.Generic;
+using BPEssentials.Enums;
+using BPEssentials.ExtensionMethods;
+using BrokeProtocol.API.ExtensionMethods;
+using BrokeProtocol.Collections;
+using BrokeProtocol.Entities;
+
+
+namespace BPEssentials.Utils
+{
+    public static class ChatUtils
```

Comment:
I suggest changing this to `Chat`.
```suggestion
    public static class Chat
```

Reply:
The Enums are also called Chat, so I would prefer to keep this name